portal/settings.py

```diff
@@ -276,3 +276,9 @@
 
 MEDIA_URL = "/media/"  # URL to serve media files
 MEDIA_ROOT = os.path.join(BASE_DIR, "media")  # Local filesystem path
+
+# Google Drive Settings
+GOOGLE_SERVICE_ACCOUNT_FILE = os.path.join(
+    BASE_DIR, "krishnapachauri-208910-574aa1e639b3.json"
+)
+GOOGLE_DRIVE_FOLDER_ID = "18P_t0wpEXuMQyQkZJxccsacOh-eelLDk"
```

Comment:
Use env var instead of hardcoding

Reply:
Yeah, I’ve make it a test version to verify the logic is correct or not, will fix it once we agree that this is what we gonna do.